src/components/Calender/CalendarCard.tsx

```diff
@@ -12,7 +12,7 @@ interface Props {
     onPressScroll: (index) => void;
     cardIndex: number;
 }
-
+//unused apparently
```

Comment:
unless you have a specific future use in mind, it might be worth deleting this dead code

Reply:
ok... its just hard to let my code babies go :(